### ISSUE TITLE
Screenreader announces the full day name (VAMC)

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#21731]
## Issue Description

This issue was originally reported in #6025. The shorthand pattern for expressing hours of operation like **Mon:** 24/7 to represent Monday, 24 hours a day, 7 days a week is not made clear to assistive technology. The markup for both day and time should be improved but this issue will address only the day of the week from the template perspective. Hours are sent via API and need to be handled differently. 

## Task: 
- [ ] Update markup to better serve screen readers. A code snippet has been provided by Trevor below. 
```
<span aria-hidden="true">Mon:</span>
      <span class="vads-u-visibility--screen-reader">Monday</span>
```

## Type of Issue
template

## Acceptance Criteria
* [ ] Screen reader announces the full day name

## Screenshots or Trace Logs

<img width="657" alt="Screen Shot 2020-02-19 at 4 31 54 PM" src="https://user-images.githubusercontent.com/934879/74883339-5e151200-5336-11ea-8723-743807338d1e.png">

**Comment by mmiddaugh** [comment]
Suggest having this reassessed as part of [Accessibility Production Review [Facilities Team - VAMC pages](https://github.com/department-of-veterans-affairs/va.gov-team/issues/36974)
